src/server.cpp

```diff
@@ -466,6 +465,14 @@ CServer::CServer ( const int          iNewMaxNumChan,
        SIGNAL ( SvrRegStatusChanged() ),
        this, SLOT ( OnSvrRegStatusChanged() ) );
 
+    QObject::connect ( QCoreApplication::instance(),
```

Comment:
Is this called when the server GUI is closed?

Reply:
Yes, tested on Linux and Windows GUI mode.